provisioner/influx-series.js

```diff
@@ -39,6 +39,78 @@ module.exports.spotRequestsSubmitted = new base.stats.Series({
   },
 });
 
+// Store the times when a spot request is fulfilled
+module.exports.spotRequestsFulfilled = new base.stats.Series({
```

Comment:
How about just doing: `exports.spotRequestsFulfilled` that goes to the entire file. Or use es6 style export like `export var spotRequestsSubmitted = ...` or even
```js
var spotRequestsFulfilled = ...

export {spotRequestsSubmitted, spotRequestsFulfilled}
```
Also `'use strict'` is not needed with babeljs.

Just two random things.. `module.exports` is a bit disturbing here.